packages/ckeditor5-list/tests/listformatting/listitemfontfamilyintegration.js

```diff
@@ -0,0 +1,333 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph.js';
+import BlockQuoteEditing from '@ckeditor/ckeditor5-block-quote/src/blockquoteediting.js';
+import CodeBlockEditing from '@ckeditor/ckeditor5-code-block/src/codeblockediting.js';
+import HeadingEditing from '@ckeditor/ckeditor5-heading/src/headingediting.js';
+import TableEditing from '@ckeditor/ckeditor5-table/src/tableediting.js';
+import FontFamilyEditing from '@ckeditor/ckeditor5-font/src/fontfamily/fontfamilyediting.js';
+import VirtualTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor.js';
+import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils.js';
+import { setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model.js';
+import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view.js';
+
+import stubUid from '../list/_utils/uid.js';
+import ListEditing from '../../src/list/listediting.js';
+import ListItemFontFamilyIntegration from '../../src/listformatting/listitemfontfamilyintegration.js';
+
+describe( 'ListItemFontFamilyIntegration', () => {
+	let editor, model, view;
+
+	testUtils.createSinonSandbox();
+
+	beforeEach( async () => {
+		editor = await VirtualTestEditor.create( {
+			plugins: [
+				ListItemFontFamilyIntegration,
+				FontFamilyEditing,
```

Comment:
We should have tests for editor without FontFamilyEditing plugin added. I'd expect code coverage is not 100% now.